metadata-service/plugin/src/main/java/com/datahub/plugins/common/SecurityMode.java

```diff
@@ -0,0 +1,41 @@
+package com.datahub.plugins.common;
+
+import java.io.FilePermission;
+import java.net.SocketPermission;
+import java.nio.file.Path;
+import java.security.AllPermission;
+import java.security.Permissions;
+import java.util.function.Function;
+
+
+public enum SecurityMode {
```

Comment:
Let's make sure this class has a test

Reply:
Please add this test

Reply:
It is covered in TestPluginPermissionManager